ios/Sources/AboutFeature/AboutFeature.swift

```diff
@@ -0,0 +1,31 @@
+import ComposableArchitecture
+
+public struct AboutState: Equatable {
+    public var staffs: [Staff]
+    public var contributors: [Contributor]
+    public var selectedType = SelectedType.staff
```

Comment:
Please add this to initializer argument and set default value to `SelectedType.staff` 🙏🏼 

It can be easy to debug with Xcode Preview!